test/deploy.sh

```diff
@@ -72,7 +72,6 @@ function usage {
 USAGE: $ME [OPTIONS] config_file_name [extra args to node]
 
    -h             print this help message
-   -D duration    test run duration (default: $DURATION)
```

Comment:
Need to keep duration option for Jenkins, can remove once we move over to Travis + docker to test PRs.